src/linker/Linker/LinkerAttributesInformation.cs

```diff
@@ -6,24 +6,38 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
This looks weird - should not be needed - please remove any unused "using" statements.